api/query.go

```diff
@@ -16,7 +16,7 @@ import (
 // Query for slice endpoints
 // @Description Query payload for slice endpoints /slice and /slice/metadata.
 type SliceQuery struct {
-	// The blob path to a vds on form: container/subpath
+	// The blob path to a vds off form: container/subpath
```

Comment:
The switch was on, now you turned it off? :smile: 

Maybe `in form` then would be better than `of form`?